ert3/storage/_storage.py

```diff
@@ -182,15 +286,15 @@ def get_ensemble_record_names(
         experiment_name = f"{workspace}.{_ENSEMBLE_RECORDS}"
     experiment = _get_experiment_by_name(experiment_name)
     if experiment is None:
-        raise KeyError(
+        raise ert3.exceptions.NonExistantExperiment(
             f"Cannot get record names of non-existing experiment: {experiment_name}"
         )
 
     ensemble_id = experiment["ensembles"][0]  # currently just one ens per exp
     response = requests.get(url=f"{_STORAGE_URL}/ensembles/{ensemble_id}/records")
     if response.status_code != 200:
         raise ert3.exceptions.StorageError(response.text)
-    return list(response.json().keys())
+    return list(x for x in response.json().keys() if not x.endswith(_METADATA))
```

Comment:
maybe a bit more verbose?

Reply:
This was remnants from the implementation in the first commits! Removed it ✅